lib/sprockets/base.rb

```diff
@@ -19,7 +19,7 @@ module Sprockets
 
   class DoubleLinkError < Sprockets::Error
     def initialize(parent_filename:, logical_path:, last_filename:, filename:)
-      message = String.new
+      message = +""
```

Comment:
In the docs it [says something about `String.new` not being 100% equivalent to `+""`](http://www.rubocop.org/en/latest/cops_performance/#performanceunfreezestring). Not sure if it applies here?

Reply:
No. But beyond that there's really no reason to concatenate a string in this fashion here, instead do:

```ruby
message = <<~MSG
  Multiple files with the same output path cannot be linked (#{logical_path.inspect})
  In #{parent_filename.inspect} these files were linked:
    - #{last_filename}
    - #{filename}
MSG
```

Reply:
I even [spared a local variable](https://github.com/rails/sprockets/pull/594/commits/f1ce104144f6d22ce3cae7f56eeffb7e006fa859)! :smile: Thanks!!